synapse/rest/client/v1/login.py

```diff
@@ -44,6 +45,7 @@ class LoginRestServlet(RestServlet):
     TOKEN_TYPE = "m.login.token"
     JWT_TYPE = "org.matrix.login.jwt"
     JWT_TYPE_DEPRECATED = "m.login.jwt"
+    APPSERVICE_TYPE = "uk.half-shot.msc2778.login.application_service"
```

Comment:
Should this be included in the `GET` requests (when `flows` is returned)?

Reply:
Yes https://github.com/matrix-org/matrix-doc/pull/2778#discussion_r489605861

Reply:
sure, done.